### DISCUSSION TITLE
Maestral continues to crash in background

### DISCUSSION THREAD

**Discussion by schwerd** [samschott/maestral#938] [General]
I've brought this up several times over the past few months. it seems to be intermittent, which doesn't make a whole lot of sense.  I've submitted crashlogs and activity snapshots.  Maestral crashes pretty regularly.  at least several times a day. I've watched and most often it's a complete crash of the UI and the daemon.  and it very often happens when my laptop is closed, therefore asleep, but not shutdown.  I will open the laptop to work, and Maestral is gone.

also, I'm watching it on mulitple OS's.  it is crashing on release version of Ventura as well as dev beta 2 of Sonoma

I'm happy to submit more logs or snapshots to discover possible causes.

I can't believe I'm the only on having these problems.  if they were really sporadic and inconsequential, I'd live with it.  but it's happening several times each day on multiple machines and causing a lot of confusion regarding shared data.  and, it seemed to work flawlessly for a long time.  so I don't know what could have changed.

**Comment by theandrewwade** [comment]
Me too. Happens on my machine and other's machines. I have a checklist below "check mail" and "check Slack" to "check maestral running" 😕

**Reply by theandrewwade** [reply]
Machines running latest release versions of Ventura and Monterey btw. No Sonoma betas.

**Comment by schwerd** [comment]
update August 13, 2023.  after updating to  1.8.0 it appears the crashing has substantially stopped.  I am looking across machines running both Ventura released version and one running Sonoma public beta.  anecdotally, things seem much more stable and I'm not seeing the crashing.  but my schedule has settled down a little now, and I'll be able to watch and track it more carefully rather just haphazardly.

thank you for all the support and feedback.  I think (and hope) that Sam appreciates our feedback.  I think he has a day job so Maestral is a labor of love.....which many of us have now come to rely on.  so thank you to Sam.
